modules/admin-top-bar/module.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Elementor\Modules\AdminTopBar;
+
+use Elementor\Core\Base\App as BaseApp;
+use Elementor\Core\Experiments\Manager;
+use Elementor\Plugin;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseApp {
+
+	/**
+	 * @return bool
+	 */
+	public static function is_active() {
+		return is_admin();
+	}
+
+	public static function get_experimental_data() {
+		return [
+			'name' => 'admin-top-bar',
+			'title' => __( 'Admin Top Bar', 'elementor' ),
+			'description' => __( 'Adds a top bar to elementors pages in admin area.', 'elementor' ),
+			'release_status' => Manager::RELEASE_STATUS_ALPHA,
+			'default' => Manager::STATE_INACTIVE,
+		];
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_name() {
+		return 'admin-top-bar';
+	}
+
+	private function render_admin_top_bar() {
+		?>
+		<div id="e-admin-top-bar">
+		</div>
+		<?php
+	}
+
+	/**
+	 * Enqueue admin scripts
+	 */
+	private function enqueue_scripts() {
+		$direction_suffix = is_rtl() ? '-rtl' : '';
+
+		wp_register_style( 'elementor-admin-top-bar', $this->get_css_assets_url( 'admin-top-bar' . $direction_suffix ), [], ELEMENTOR_VERSION );
+
+		wp_enqueue_style( 'elementor-admin-top-bar' );
+
+		wp_enqueue_script( 'elementor-admin-top-bar', $this->get_js_assets_url( 'admin-top-bar' ), [
+				'react',
+				'react-dom',
+			], ELEMENTOR_VERSION, true );
+
+		$min_suffix = Utils::is_script_debug() ? '' : '.min';
+
+		wp_enqueue_script( 'tipsy', ELEMENTOR_ASSETS_URL . 'lib/tipsy/tipsy' . $min_suffix . '.js', [
+				'jquery',
+			], '1.0.0', true );
```

Comment:
Something wrong with the indentation here

Reply:
Done.